### PR TITLE
lab-supervised-learning-sklearn - Gabriela Scatena

### PR DESCRIPTION
**Pull request by GabrielaScatena**
Without bonus - but with doubts ;)

### CODE REVIEW

**Comment by ta-data-lis** [comment]
Lab checked. Well done!

```
r2_score(y_train09, y_test_pred)
```
This is throwing an error because you're trying to compare the labels of the train set against the labels of the test set, which has a different size.

Keep up the good work! 👍
